definitions/ext-gatsby_build/definition.yml

```diff
@@ -0,0 +1,20 @@
+domain: EXT
+# Derived from the Fastly log ingestion, a datacenter is a Point of Presence
+# https://docs.fastly.com/en/guides/fastly-pop-locations
+type: GATSBY_BUILD
+
+synthesis:
+  name: gatsbySite
+  identifier: gatsbySite
+  encodeIdentifierInGUID: false
```

Comment:
We probably want this to true, I assume the gatsbySite is likely to be potentially over 100 chars?

Reply:
I'm not sure if it ever would be, this is set manually, unless i'm misunderstanding the use of `identifier`

Reply:
Ok cool i'll set `encodeIdentifierInGUID: true`. Ye so after sending those traces, I wanted to refactor the attributes a little, so thought `gatsbySite` might be a good unique identifier, i'll run a trace now to test that first though